ouroboros-consensus-cardano/src/Ouroboros/Consensus/Cardano/Orphans.hs

```diff
@@ -0,0 +1,187 @@
+{-# LANGUAGE DeriveAnyClass             #-}
+{-# LANGUAGE DerivingStrategies         #-}
+{-# LANGUAGE DerivingVia                #-}
+{-# LANGUAGE FlexibleContexts           #-}
+{-# LANGUAGE FlexibleInstances          #-}
+{-# LANGUAGE GeneralizedNewtypeDeriving #-}
+{-# LANGUAGE LambdaCase                 #-}
+{-# LANGUAGE MultiParamTypeClasses      #-}
+{-# LANGUAGE OverloadedStrings          #-}
+{-# LANGUAGE RankNTypes                 #-}
+{-# LANGUAGE ScopedTypeVariables        #-}
+{-# LANGUAGE StandaloneDeriving         #-}
+{-# LANGUAGE TypeApplications           #-}
+{-# LANGUAGE TypeFamilies               #-}
+{-# LANGUAGE TypeOperators              #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
+
+{-# LANGUAGE NamedFieldPuns             #-}
```

Comment:
Move this up to the upper group, please.

Reply:
Hmm, actually, is it even being used?

I don't see any standalone deriving etc either, so maybe the whole set of LANGUAGE could use a trim.